frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/tabs/details-resources-requirement.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { testName } from '../../support';
+import { K8S_KIND, VM_STATUS, YAML_VM_NAME } from '../../utils/const/index';
+import {
+  detailsTab,
+  modalConfirm,
+  modalTitle,
+  pendingChangeAlert,
+  saveAndRestart,
+} from '../../views/selector';
+import { tab } from '../../views/tab';
+import { waitForStatus, vm } from '../../views/vm';
+
+const flavor = '[data-test-id="details-Flavor"]';
+const flavorString = 'Tiny: 1 CPU | 1.5 GiB Memory';
+const flavorCustomString = 'Custom: 2 CPU | 3 GiB Memory';
+const flavorSelect = '#vm-flavor-modal-flavor';
+const customCPU = '#vm-flavor-modal-cpu';
+const customMem = '#vm-flavor-modal-memory-size';
+
+const dedicatedResources = '[data-test-id="details-Dedicated Resources"]';
+const dedicatedResourcesCheckbox = '#dedicated-resources-checkbox';
+const dedicatedResourcesString = 'Workload scheduled with dedicated resources (guaranteed policy)';
```

Comment:
It makes sense to keep ones that are specific to a test in the test, but I think we should keep strings that are used in the UI in a strings file(s) in case they need to be reused.

Reply:
Move selectors and strings into separated files.